packages/babel-generator/test/fixtures/importAttributesKeyword/assertions-with-to-default/output.js

```diff
@@ -1 +1 @@
-import "a" with type: "json";
\ No newline at end of file
+import "a" with { type: "json" };
```

Comment:
We output the standard syntax on input `import "a" with { type: "json" };` and options
```json
{
  "BABEL_8_BREAKING": false,
  "plugins": ["importAssertions"]
}
```